spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderPropertiesAccessor.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author Gary Russell
  */
-public abstract class AbstractBinderPropertiesAccessor implements BinderProperties {
+public abstract class AbstractBinderPropertiesAccessor {
```

Comment:
Yeah, I believe it is hard to distinguish most of these properties whether they are `binding` or `binder` properties and they mostly belong to the same. I like the idea of separating them based on `producer/consumer` properties. Also, it would be better we name these as `producer/consumer` **binding** properties rather than **binder** properties since `binder` properties can only be [this] (https://github.com/ilayaperumalg/spring-cloud-stream/blob/per-binding-properties-fix/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BinderProperties.java)